test/test_helper.js

```diff
@@ -0,0 +1,4 @@
+process.env.NODE_ENV = 'test'
+require('../app.js')
```

Comment:
@nonword & spoke: We're not sure what was going on with his localhost earlier - but we see that running tests with a dev env up - loudly will throw a `listen EADDRINUSE :::3000` error.

We're going to merge in this PR - and follow up with a ticket to configure another port for tests.